### ISSUE TITLE
update curriculum planning documents to reflect new "planning-weeks.md" process

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/master-reference#280]
## Proposal

Using [this pattern format](https://github.com/foundersandcoders/a-pattern-language/issues/7)

#### Problem:

FAC9 were instructed to use one issue for all the curriculum planning for their week. This resulted in one of the following:
+ a huge list of comments on each issue, which no one is going to read through in future
+ or mentors opened a (sometimes private) gitter channel for their discussions instead
  + all this curriculum planning was therefore entirely unrecorded
  + very difficult to maintain the `master-reference`, because the first time they are aware of decisions that were made on gitter, is through the PR. By this point, mentors believed their week to be finished and requesting major changes was understandably frustrating for mentors.

#### Solution:

+ Course Facilitator to open a single issue for each week and assign the `week` label (to indicate that this is a week-level epic) and the `week-<x>` label (to indicate whether it is node week or toolkit week, etc)
+ Students use this issue as an "epic" and create sub-issues from it, each with the `week-<x>` label
+ Students then assign someone from the team to each task & add time estimates

#### Hypotheses:

+ Course Facilitator acting as Scrum Master of the process will unblock teams (empower them to make the necessary changes)
  + Creating a backlog of issues makes it visually clear how much work there is to do.
  + Assigning themselves to each issue makes it visually clear who is doing more/less of the work
  + Assigning labels effectively will make it easier for Course Facilitators to maintain the `master-reference`
  + Adding time estimates and reviewing with the Course Facilitator will ensure that there are less surprises right before a week is supposed to be delivered

#### Progress so far

[Some teams in FAC10 started this during their curriculum planning week](https://github.com/foundersandcoders/master-reference/issues/280#issuecomment-305801204).

I have introduced this more formally during [one-on-one meetings](https://github.com/foundersandcoders/master-reference/issues/324) with each group of mentors, but haven't quite managed time estimates yet
+ [Toolkit week](https://github.com/foundersandcoders/master-reference/issues/311)
+ [Testing week](https://github.com/foundersandcoders/master-reference/issues/299)

---

### Original top comment
The process by which the current cohorts will plan the next curriculum is still evolving. The current ["planning weeks" doc](https://github.com/foundersandcoders/master-reference/blob/master/curriculum-planning/planning-weeks.md) refers to the process used by FAC9 for creating the `master-reference` from scratch. The current cohorts need a new and different process. Considering the large structural changes that have been suggested, it may not be as simple as just dividing the cohort into separate weeks.

At the moment, @bradreeder and I have discussed the formation of 2 task groups:
+ improving research day
+ creating the full set of 1-hour HTML & CSS workshops

As the process is currently being formed on the fly, editing this document beforehand has proven unrealistic.

We need to document the process that gets used over the following few days, as well as any potential alterations that we would make to this process, for the next set of cohorts.

**Comment by jsms90** [comment]
A useful pattern that is emerging from FAC10's curriculum planning week is the idea of having one `epic`-style issue that details all the plans for changes. This makes complete sense, as it mirrors the way that we plan our software sprints.

Here are some good examples:
+ node week 1 #186 
+ postgres week #32

**Comment by des-des** [comment]
@jsms90 :+1: I have opened up an issue to address this point more specifically: https://github.com/foundersandcoders/master-reference/issues/316

**Comment by eliascodes** [comment]
I'm happy with this approach in general. Unsure of how much value time estimates will bring though, tbh.

**Comment by jsms90** [comment]
Personally, I think they would bring about as much value as you believe time estimates ever bring to a project i.e. it depends on how much the devs (in this case mentors) value the process and how effective the Scrum Master (in this case Course Facilitator) is at their role.

Absolutely zero comment on how effective I consider them to be, given that I've never actually used them myself :flushed: (yet)

**Comment by des-des** [comment]
@jsms90 I know when I was course facilitating I would not have had time to do this kind of scrum mastering.

Also, time estimating dev work is a very particular skill, different to time estimating work in general. Estimating dev work is also essential to managing the financial cost of a project. In this case students are doing voluntary work.

**Comment by eliascodes** [comment]
Week epic issues have been used for the past two rounds of curriculum planning and are likely to continue to feature. The 'planning weeks' doc has been removed as it was no longer relevant.
